app/workers/postcodes_collection_worker.rb

```diff
@@ -0,0 +1,18 @@
+class PostcodesCollectionWorker
+  include Sidekiq::Worker
+
+  sidekiq_options queue: :collect_postcodes
+
+  def perform(test: false)
+    loop do
+      postcodes = Postcode.select(:postcode)
```

Comment:
Just doing a quick bit of arithmetic on this: what kind of performance issues might we hit by selecting 12 million rows? 🤔 

Assuming each postcode is about 7 characters long, therefore 7 bytes, this query might put 84 million bytes in memory, i.e. 82031 KB or 80.1 MB. Shouldn't be an issue.

But as a quick test, I spun up a Rails console in Whitehall and did a `Edition.select(:document_id)`, which [has an index](https://github.com/alphagov/whitehall/blob/5d45cad7c2606f0fc0fa29d784de2bc7c9bd8fe9/db/schema.rb#L423) like [our postcode field does](https://github.com/alphagov/locations-api/blob/e7d750118d362e519704afb8c94717473845721d/db/schema.rb#L23). The query took a few minutes and ended with `(Object doesn't support #inspect) => ` - not encouraging.

So I tried `Document.select(:id)`. Again, it took a minute or two. Ended up getting 378378 results. That's 31x fewer results than we'll have in Locations API - which does make me wonder if we might be stuck for around half an hour retrieving all the rows with this `select` query!

That's probably _not_ the case - I doubt the memory allocation is that linear - but even if it was, I don't think that would necessarily be an issue either. [Long-running jobs are a thing](https://stackoverflow.com/questions/58353386/sidekiq-jobs-maximum-run-time), so it's not like Sidekiq would crash.

TLDR: I thought we may have a performance issue on this line, but on closer inspection, I don't think we do 😄